contribs/gmf/src/filters/filterselectorcomponent.js

```diff
@@ -1,33 +1,44 @@
-goog.provide('gmf.filterselectorComponent');
+goog.provide('gmf.filters.filterselectorComponent');
 
 goog.require('gmf');
 /** @suppress {extraRequire} */
 goog.require('gmf.authentication.Service');
 goog.require('gmf.datasource.DataSourceBeingFiltered');
 /** @suppress {extraRequire} */
 goog.require('gmf.datasource.DataSourcesHelper');
-goog.require('gmf.SavedFilters');
+goog.require('gmf.datasource.OGC');
+goog.require('gmf.filters.SavedFilters');
 /** @suppress {extraRequire} */
 goog.require('ngeo.filter.module');
 /** @suppress {extraRequire} */
 goog.require('ngeo.message.modalComponent');
 goog.require('ngeo.message.Notification');
+goog.require('ngeo.message.Message');
 /** @suppress {extraRequire} */
 goog.require('ngeo.filter.RuleHelper');
 goog.require('ol.events');
+goog.require('ol.Observable');
+goog.require('ol.array');
 
 goog.require('ngeo.map.FeatureOverlayMgr');
 
 
-// In the future module declaration, don't forget to require:
-// - ngeo.map.FeatureOverlayMgr.module.name
-// - ngeo.message.Notification.module.name
-// - ngeo.message.modalComponent.name
+gmf.filters.filterselectorComponent = angular.module('gmfFilterselector', [
+  gmf.authentication.Service.module.name,
+  ngeo.map.FeatureOverlayMgr.module.name,
+  ngeo.message.Notification.module.name,
+  ngeo.message.modalComponent.name,
+  ngeo.filter.module.name,
```

Comment:
We won't load the whole module in a component. It can evolve with think we don't want in this component.

Reply:
Which one do you mean here? `ngeo.filter.module.name`?

Reply:
yes

Reply:
Changed it